R/variables.R

```diff
@@ -415,17 +427,27 @@ calculate_initial_counts <- function(parameters) {
 
 calculate_eq <- function(het_nodes, parameters) {
   ft <- sum(get_treatment_coverages(parameters, 0))
-  lapply(
-    het_nodes,
-    function(n) {
-      malariaEquilibrium::human_equilibrium_no_het(
-        parameters$init_EIR * calculate_zeta(n, parameters),
-        ft,
-        parameters$eq_params,
-        EQUILIBRIUM_AGES
-      )
-    }
-  )
+  if(parameters$parasite == "falciparum"){
+    lapply(
+      het_nodes,
+      function(n) {
+        malariaEquilibrium::human_equilibrium_no_het(
+          parameters$init_EIR * calculate_zeta(n, parameters),
+          ft,
+          parameters$eq_params,
+          EQUILIBRIUM_AGES
+        )
+      }
+    )
+
+  } else if (parameters$parasite == "vivax"){
+    eq <- malariaEquilibriumVivax::vivax_equilibrium(
```

Comment:
Can you add malariaEquilibriumVivax to DESCRIPTION? In the same way malariaEquilibrium is

Reply:
Added in the next commit.